subprojects/core/src/main/java/org/gradle/internal/scan/config/BuildScanPluginCompatibility.java

```diff
@@ -18,13 +18,14 @@
 
 import org.gradle.util.VersionNumber;
 
-class BuildScanPluginCompatibility {
+public class BuildScanPluginCompatibility {
 
-    public static final VersionNumber MIN_SUPPORTED_VERSION = VersionNumber.parse("2.0.2");
-    private static final String MIN_SUPPORTED_VERSION_DISPLAY = "2.0.2";
-    public static final String UNSUPPORTED_PLUGIN_VERSION_MESSAGE =
-        "This version of Gradle requires version " + MIN_SUPPORTED_VERSION_DISPLAY + " of the build scan plugin or later.\n"
-            + "Please see https://gradle.com/scans/help/gradle-incompatible-plugin-version for more information.";
+    public static final String FIRST_GRADLE_ENTERPRISE_PLUGIN_VERSION_DISPLAY = "3.0";
+    public static final VersionNumber FIRST_GRADLE_ENTERPRISE_PLUGIN_VERSION = VersionNumber.parse(FIRST_GRADLE_ENTERPRISE_PLUGIN_VERSION_DISPLAY);
+
+    public static final String OLD_SCAN_PLUGIN_VERSION_MESSAGE =
+        "The build scan plugin is not compatible with this version of Gradle.\n"
+            + "Please see https://gradle.com/help/gradle-6-build-scan-plugin for more information.";
```

Comment:
Where will this redirect? To a new section in the build scan plugin manual?

Reply:
We'll need to set up a redirect to a new section, yes.

Reply:
Ok. Added a task to https://github.com/gradle/dotcom/issues/3980